Sources/TiqrCoreObjC/Classes/AboutViewController.m

```diff
@@ -42,23 +42,14 @@ @interface AboutViewController ()
 @property (nonatomic, strong) IBOutlet UIButton *appLogo;
 @property (weak, nonatomic) IBOutlet UIButton *surfLogo;
 
-@property (nonatomic, assign) enum VersionInfo versionInfo;
-
 @end
 
-enum VersionInfo {
```

Comment:
I would call it something like VersionInfoDetailLevel

Reply:
Red means it got deleted.

Reply:
ok wait